boptimol/GP_MODEL.py

```diff
@@ -0,0 +1,239 @@
+from typing import List, Optional
+import torch
+import gpytorch
+import numpy as np
+
+from botorch.utils import standardize
+from botorch.optim import optimize_acqf
+from botorch.acquisition.analytic import *
+from botorch.models import SingleTaskGP
+from botorch.fit import fit_gpytorch_mll
+from botorch.acquisition import qKnowledgeGradient
+from gpytorch.mlls import ExactMarginalLogLikelihood
+from gpytorch import kernels as gpykernels
+from gpytorch import means as gpymeans
+
+from boptimol.molecule import Molecule
+from boptimol.mean_potentials import MorseMean, QuadraticMean
+
+class Bond_Lengths(gpytorch.models.ExactGP):
```

Comment:
I'd like to see a lot more docstrings so that others can understand what the classes do, etc.